explore/src/main/scala/explore/observationtree/ObsList.scala

```diff
@@ -301,13 +300,9 @@ object ObsList:
                       props.obsExecutionTimes.getPot(obsId).map(_.programTimeEstimate),
                       ObsBadge.Layout.ObservationsTab,
                       selected = selected,
-                      setStatusCB = obsEditStatus(obsId)
+                      setStateCB = obsEditStatus(obsId)
```

Comment:
Do we want to rename to `obsEditState` too?

Reply:
I'll do that after I rebase on your PR. 👅